convoluving_response.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import scipy.stats
+from scipy.stats import gamma
+from stimuli import events2neural
+
+def hrf(times):
+    """ Return values for HRF at given times """
+    # Gamma pdf for the peak
+    peak_values = gamma.pdf(times, 6)
+    # Gamma pdf for the undershoot
+    undershoot_values = gamma.pdf(times, 12)
+    # Combine them
+    values = peak_values - 0.35 * undershoot_values
+    # Scale max to 0.6
+    return values / np.max(values) * 0.6
+
+def constructing_convo(fname, n_volx):
+	tr_times = np.arange(0, 30, 2.5)
+	hrf_at_trs = hrf(tr_times)
+	neural_prediction = events2neural(fname + '_cond.txt', 2.5, n_volx)
```

Comment:
I just realized we can comment code! That's pretty cool. Anyways, you need to change the input file names. The cond.txt files for our data under the model/model001/onsets are named like cond001.txt